osgar/tools/lidarview.py

```diff
@@ -10,7 +10,7 @@
 import pygame
 from pygame.locals import *
 
-from osgar.logger import LogReader, lookup_stream_id, lookup_stream_names
+from osgar.logger import LogReader, LogIndexedReader, lookup_stream_id, lookup_stream_names
```

Comment:
do we need both Reader-s?

Reply:
I was suggesting to remove almost everything, so... no, we don't need both. Just that there is a lot more code involving LogReader to delete.

Reply:
OK feel free to remove the most of the code if functionality remains. Maybe also ignore the legacy code.